### ISSUE TITLE
showUserLocation without focusing?

### ISSUE THREAD

**Issue by jkallunki** [lelandrichardson/react-native-maps#110]
Is it somehow possible to show user's location without focusing to it initially? Of course it can be done by using geolocation API manually and drawing a custom marker, but I think this would still be a nice feature.

**Comment by bodyflex** [comment]
:+1:

**Comment by mrpatiwi** [comment]
+1

**Comment by heydabop** [comment]
Use the `initialRegion` prop.

**Comment by bodyflex** [comment]
@heydabop at least on iOS `initialRegion` is ignored and the user location is automatically focused on when `showsUserLocation` is `true`

**Comment by ghamaide** [comment]
You can change this by commenting this line : `_followUserLocation = showsUserLocation;` line 178 of AirMap.m file

**Comment by bodyflex** [comment]
apparently Android doesn't do this by default. I think that iOS should also not automatically focus on the users location, since it's pretty trivial to set focus manually and you'd have more control anyway.

**Comment by joshbedo** [comment]
Is there any way to show a blue dot displaying where the user is without custom markers?

**Comment by mathiasnovas** [comment]
React Native has added a prop to disable `followUserLocation`. Commit: https://github.com/wildlifela/react-native/commit/4aa43223ccfcff1a5df242e94789482867b9b5ee

**Comment by npomfret** [comment]
Is there a trick to getting showsUserLocation to work on android?  For some reason I don't see it on the map.  Also, _navigator.geolocation.getCurrentPosition_ doesn't work for me.  I've got the permissions _android.permission.ACCESS_FINE_LOCATION_ and _android.permission.ACCESS_COARSE_LOCATION_ in my manifest file... but no gps.

**Comment by jrichardlai** [comment]
@npomfret are you using genymotion? make sure you enable the location of the user if you are using the simulator.

**Comment by npomfret** [comment]
Thanks for the reply.  I've fixed my problem, well most of it. I'm using a physical device and the version of Google Play Services was not correct.  I've submitted a PR with the things I've learnt.

All that's wrong now is that I can't get the users current location in android.  I'm getting a timeout error when calling _navigator.geolocation.getCurrentPosition_.  No idea what the problem is.  I can the users location on a map, just not from the API.
